inst/shinyexample/R/run_app.R

```diff
@@ -1,11 +1,23 @@
 #' Run the Shiny Application
 #'
+#' @param onStart optional, described in ?shiny::shinyApp
+#' @param options optional, described in ?shiny::shinyApp
+#' @param enableBookmarking optional, described in ?shiny::shinyApp
+#' @param ... arguments to pass to golem_opts
+#'
 #' @export
 #' @importFrom shiny shinyApp
 #' @importFrom golem with_golem_options
-run_app <- function(...) {
+run_app <- function(onStart = NULL,
+                    options = list(), 
+                    enableBookmarking = NULL,
+                    ...) {
```

Comment:
We should also have `options` here (from `shinyApp`)

Reply:
Not  sure what you mean.  Line 12 there?

Reply:
Yep, original shinyApp function has an `options` parameter, we should be able to use it from run_app() too